docs/pages/database-access/guides/mongodb-atlas.mdx

```diff
@@ -185,8 +185,8 @@ You can then use `/tmp/isrgrootx1.pem` as the value of the `db_service.databases
 
 ## Step 3/4. Configure Atlas
 
```

Comment:
Now that there are two authentication options, I think we need a brief paragraph explaining that these entail and when you would choose each one. Otherwise, a reader needs to read both `TabItems` to understand which one they should use.